### DISCUSSION TITLE
绳网社区交流公约

### DISCUSSION THREAD

**Discussion by nostalgiatan** [share121/inter-knot#385] [其他]
1、不得在论坛内挑衅引发骂战，恶意攻击其他游戏的玩家，不得在正常讨论中故意讨论与该贴无关的内容，破坏帖子氛围。
2、不得讨论政治话题，严禁调侃、谩骂、攻击国家领导人、对国家突发事散布谣言、发表谣言或不实言论、转载非法媒体信息等非法行为。
3、禁止带节奏、挑拨其他玩家、阴阳怪气、嘲讽引战，不得对他人进行人身攻击，不得毫无证据对他人进行诽谤、造谣。
4、不得讨论违禁的黄、赌、毒话题，尤其是种子、链接、色情音像制品截图以及过于暴露的内容等。
5、不得发布带有商业广告性质的帖子或广告链接、出现二维码
6、禁止发布人身攻击、恶意辱骂，禁止恶意攻击其他网友家人、朋友、民族、宗教以及地域黑的言论

若有发布以上相关言论，管理员直接做删除处理

**Comment by ShangHuaWenYuan** [comment]
[一❤个❤人❤在❤家❤看❤伊❤埃❤斯❤热❤舞](https://www.bilibili.com/video/BV1HT421k7Rp/?spm_id_from=333.337.search-card.all.click&vd_source=9dbf0038cac01a66dbf38722f8cdaf31)

**Comment by kano0x0** [comment]
本绳匠复议，建议服主置顶加大该贴

**Comment by cow-your-sister** [comment]
管理在哪？除了仓库owner我看在GitHub上谁能删的了帖子

**Reply by cow-your-sister** [reply]
有本事你把我这条删了先

**Reply by nostalgiatan** [reply]
目前也就只有他一个人是管理了

**Comment by Rol17** [comment]
支持  营造良好的社区氛围需要大家共同努力
